references/classification/presets.py

```diff
@@ -16,8 +16,16 @@ def __init__(
         ra_magnitude=9,
         augmix_severity=3,
         random_erase_prob=0.0,
+        backend="pil",
     ):
-        trans = [transforms.RandomResizedCrop(crop_size, interpolation=interpolation)]
+        trans = []
+        backend = backend.lower()
+        if backend == "tensor":
+            trans.append(transforms.PILToTensor())
+        else:
+            assert backend == "pil"
```

Comment:
Nit: make it a little more expressive?

```suggestion
        elif backend != "pil":
            raise ValueError("backend can be 'tensor' or 'pil', but got {backend}")
```